src/vcsparser.core/ChangesetProcessor.cs

```diff
@@ -69,6 +76,19 @@ private void ProcessFileChange(IChangeset changeset, bool containsBugs, FileChan
             }
         }
 
+        private void ProcessBugDatabaseFileChange(IChangeset changeset, bool containsBugs, FileChanges c)
+        {
+            var fileName = GetFileNameConsideringRenames(c.FileName);
+
+            DailyCodeChurn dailyCodeChurn = FindOrCreateDailyCodeChurnForFileAndDate(changeset, fileName);
+            if (dailyCodeChurn.BugDatabase == null)
+                dailyCodeChurn.BugDatabase = new DailyCodeChurnBugDatabase();
+
+            ProcessBugDatabaseChanges(c, dailyCodeChurn);
+            if (containsBugs)

```

Comment:
Shouldn't check containsBug, since BugDatabase and parsing log message are 2 completely independent metrics.

Reply:
This also deserves a test: One changeset which log message matches the regex another changeset that doesn't match the regex but exists on the bug database.

Reply:
Removed the need for `if (containsBugs)` in 436638d125bb40b11223df403d8dd9b234d55800
Tests already existed but now tweaked for expected outcome and renamed